docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -13,40 +13,62 @@ import TabItem from '@theme/TabItem';
 
 # Semgrep Pro Engine overview
 
-## Introduction
-
 <SemgrepProEngineIntroduction extended/>
 
-## Semgrep Pro engine language support
+<br />
+<br />
 
-Refer to [Supported languages](/supported-languages/#semgrep-pro-engine) to see languages supported by Semgrep Pro Engine.
+:::info Language support
+Refer to [<i class="fa-regular fa-file-lines"></i> Supported languages](/supported-languages/#semgrep-pro-engine) to see languages supported by Semgrep Pro Engine.
+:::
 
 ## Using and running Semgrep Pro Engine
 
-This section guides you through the Semgrep Pro Engine installation and helps you to scan your projects both in CLI and with the Semgrep Cloud Platform (SCP).
+This section guides you through the Pro Engine installation and helps you to scan your projects both in CLI and with Semgrep Cloud Platform (SCP).
+
+### Running Semgrep Pro Engine in Semgrep Cloud Platform
+
+:::info Prerequisite
+- An existing Semgrep Cloud Platform account. See [<i class="fa-regular fa-file-lines"></i> Signing in to SCP](/semgrep-code/getting-started/#semgrep-code-with-semgrep-cloud-platform).
+:::
+
+This is the preferred method to run Semgrep Pro Engine. It enables you to view and triage your findings from a centralized location. Code is not uploaded.
+
+To run Semgrep Pro Engine in the Semgrep Cloud Platform, follow these steps:
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
+1. Click **[Settings](https://semgrep.dev/orgs/-/settings)**.
+1. Ensure that the <i class="fa-solid fa-toggle-large-on"></i> **Pro Engine beta** toggle is enabled.
+1. Ensure that you have the **default ruleset** added in your **[Policies page](https://semgrep.dev/orgs/-/policies)**. If this ruleset is **not** added, go to [<i class="fas fa-external-link fa-xs"></i> Semgrep Registry - Default ruleset page](https://semgrep.dev/p/default), then click **Add to Policy**. For best results, set this ruleset to the **Monitor** rule mode.
+1. Perform either of the following:
+    1. **If you don't have any repositories for scanning in Cloud Platform:** Follow the procedures in [<i class="fa-regular fa-file-lines"></i> Scanning a repository](/semgrep-code/getting-started/#semgrep-code-with-semgrep-cloud-platform) to scan a new repository with Semgrep Pro Engine. Ensure that your project's language is supported by Semgrep Pro Engine.
+    1. **If you have existing repositories in Semgrep Cloud Platform:** Full scans now include Semgrep Pro Engine. You can trigger a full scan through your CI provider.
+
+Pro Engine now runs on all full scans.
```

Comment:
Minor nit: are partial scans possible? If not, can we just say "all scans" here?

Reply:
Partial scans (PR scans) are not possible yet - they will be by December :) I wonder if it's worth it to say that for now?

Reply:
Yeah i think it's better to be explicit.